src/main/java/org/spongepowered/api/service/event/EventManager.java

```diff
@@ -34,13 +35,23 @@
     /**
      * Registers an object to receive {@link Event}s.
      *
-     * @param plugin A plugin instance
+     * @param plugin The plugin instance
      * @param obj The object
      * @throws IllegalArgumentException Thrown if {@code plugin} is not a plugin instance
      */
     void register(Object plugin, Object obj);
```

Comment:
We aren't renaming this method. We also aren't returning the handlers as it would have to be immutable and that would really do no good for you.

The better way would be to let you ask the event manager (by class) for an immutable list of eventhandlers (or just the amount of handlers registered for that event, which is what anyone really wants that for anyhow).